bl31/aarch64/crash_reporting.S

```diff
@@ -60,8 +61,11 @@ intr_excpt_msg:
 	 * Helper function to print newline to console.
 	 */
 func print_newline
+	mov	x3, x30
 	mov	x0, '\n'
-	b	plat_crash_console_putc
+	crash_console_putc
```

Comment:
I don't think this patch will be necessary if you hook into the console framework properly in the earlier patches.